index.js

```diff
@@ -93,3 +93,147 @@ for (let i = 0; i < card.length; i++) {
+        if (event.finish) {  // finish === null for instantaneous events
+            finishDate = new Date(event.finish);
+            let finishString = convertDate(finishDate);
+            if (dateString.slice(-2) === finishString.slice(-2)) {  // hide "am/pm" of first time if both are identical
+                dateString = dateString.slice(0, -2);
+            }
+            dateString += " - " + convertDate(finishDate);
+        }
+
+        // calculate event container classes
+        let divClasses = 'event';
+        if (finishDate < now) {
+            divClasses += ' event-complete';
+        }
+        else if (startDate < now && now < finishDate) {
+            divClasses += ' event-live';
+        }
+
+        // adding event to the page
+        var eventContainer;
+        // Deciding which day content box it goes inside
+        switch (startDate.getDate()) {
+            case 5: eventContainer = $('#mar-5-content'); break;
+            case 6: eventContainer = $('#mar-6-content'); break;
+        }
+        // If it doesn't fall into one of those event days, log the problem and quit trying to add it
+        if (!eventContainer) {
+            console.log("Event " + event.title + " date " + startDate + " out of range.");
+            return; // skip current iteration https://stackoverflow.com/a/31399448/1431900
+        }
+        // Building HTML and adding it to page
+        let html = `<div class="${divClasses}"><p class="time">${dateString}</p><p>${event.title}</p></div>`;
+        const eventDiv = eventContainer.append(html);
+    });
+}
+
+// Comparing two events to sort them - BH7 code unmodified
+function compareEvents(a, b) {
+    // We can sort by start/end here because the ISO 8061
+    // timestamps given by the server are lexicographically
+    // sortable.
+    if (a.start < b.start) {         // if a starts before b...
+        return -1;                   //   ...then a goes before b
+    } else if (a.start > b.start) {  // if a starts after b...
+        return 1;                    //   ...then b goes before a
+    } else {
+        if (a.end < b.end) {         // if a ends before b...
+            return -1;               //   ...then a goes before b
+        } else if (a.end > b.end) {  // if a ends after b...
+            return 1;                //   ...then b goes before a
+        } else {
+            return 0;
+        }
+    }
+}
+
+// Converting dates to something more user friendly - BH7 code unmodified
+function convertDate(date) {
+    let output = '';
+    // hour
+    if (date.getHours() > 12) {
+        output = String(date.getHours() - 12);
+    } else {
+        output = String(date.getHours());
+    }
+    // minute
+    if (date.getMinutes() !== 0) {
+        output += ':' + String(date.getMinutes()).padStart(2, '0');
+    }
+    // AM/PM
+    if (date.getHours() >= 12) {
+        output += 'pm';
+    } else {
+        output += 'am';
+    }
+    return output;
+}
```

Comment:
nit: missing newline at EOF
```suggestion
}

```